rdt-open-reader/src/covidIndonesia/assets/json.form/patient-registration-form-original.json

```diff
@@ -273,6 +273,23 @@
         "openmrs_entity_id": "",
         "type": "edit_text"
       },
+      {
+        "key": "gvt_id_address_lbl",
+        "type": "label",
+        "text": "Government ID address?",
```

Comment:
Modify this to https://github.com/onaio/rdt-standard/issues/737#issuecomment-769777606.

Reply:
text updated.